client/src/components/shared/Layout/LayoutNavbar/LayoutNavbar.tsx

```diff
@@ -58,8 +58,8 @@ const LayoutNavbar: FC<LayoutNavbarProps> = ({ navigationBottomSuffix }) => {
                 className={cx(styles.buttonNavigation, isActive && styles.isActive)}
                 dataTest={`Navbar__Button--${label}`}
                 Icon={
-                  to === ROOT_ROUTES.settings.absolute ? (
-                    <Svg img={icon} size={isTablet ? 3 : 2.4} />
+                  to === ROOT_ROUTES.settings.absolute || to === ROOT_ROUTES.allocation.absolute ? (
```

Comment:
I suggest using `isBigIcon` in the `navigationTabs` themselves. Here we have config spread out instead of being contained in one `const`.